### PR TITLE
Add READMEs to components that are missing one

### PR DESCRIPTION
**Pull request by akoushke**

### CODE REVIEW

File: src/components/WebexLocalMedia/README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Webex Local Media Component
+
+Webex local media component displays the user local stream if enabled and avatar of a _Webex Teams_ user if disabled.
```
Comment (lalli-flores) [inline src/components/WebexLocalMedia/README.md]:
```suggestion
Webex local media component displays the user local stream, if enabled. If user's local media is disabled the avatar of the current  _Webex Teams_ user is displayed.
```

File: src/components/WebexMeetingInfo/README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Webex Meeting Info Component
+
+Webex Meeting Info component displays title of the meeting.
```
Comment (lalli-flores) [inline src/components/WebexMeetingInfo/README.md]:
```suggestion
Webex Meeting Info component displays title of a meeting.
```

File: src/components/WebexRemoteMedia/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Webex Remote Media Component
+
+Webex remote media component displays the remote stream of the meeting.
```
Comment (lalli-flores) [inline src/components/WebexRemoteMedia/README.md]:
```suggestion
Webex remote media component displays the remote stream of a meeting.
```

**Comment by akoushke** [comment]
Addressed comments.

**Comment by akoushke** [comment]
> I will send you better pics for the READMEs

should be fine but sure.

File: src/components/WebexMeetingInfo/README.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Webex Meeting Info Component
+
+Webex Meeting Info component displays title of a meeting.
```
Comment (adamweeks) [inline src/components/WebexMeetingInfo/README.md]:
```suggestion
Webex Meeting Info component displays information about a meeting, including: title, start time and end time (if available).
```

**Pull request merged**